frontend/public/index.prod.html

```diff
@@ -69,5 +68,6 @@
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
     <script type="text/javascript" src="%PUBLIC_URL%/js/collapsible.js"></script>
+    <script type="text/javascript" src="%PUBLIC_URL%/js/env.js"></script>
```

Comment:
we can probably just have one index.html with this setup (instead of needing an index.prod.html and an index.html)

Reply:
Yeah I just removed `index.prod.html` and cleaned up the code.